pkgs/build-support/binutils-wrapper/setup-hook.sh

```diff
@@ -0,0 +1,63 @@
+# Binutils Wrapper hygiene
+#
+# See comments in cc-wrapper's setup hook. This works exactly the same way.
+
+binutilsWrapper_addLDVars () {
+    case $depOffset in
+        -1) local role='BUILD_' ;;
+        0)  local role='' ;;
+        1)  local role='TARGET_' ;;
```

Comment:
Where may `depOffset` be set to 1? I find only places where it is set to -1 or 0.

Reply:
Oh heh. That would anticipate https://github.com/NixOS/nixpkgs/pull/26805

Reply:
Cc-wrapper already has that, I guess I'll leave it as consistent anticipation.